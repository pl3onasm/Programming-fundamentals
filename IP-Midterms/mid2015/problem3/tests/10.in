202020 2020
