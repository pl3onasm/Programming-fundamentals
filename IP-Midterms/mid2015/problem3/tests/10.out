202020*2020=808080+6464640+12929280+25858560+51717120+103434240+206868480=408080400
